components/brave_rewards/resources/extension/brave_rewards/background/reducers/rewards_panel_reducer.ts

```diff
@@ -6,19 +6,27 @@ import { types } from '../../constants/rewards_panel_types'
 import * as storage from '../storage'
 import { getTabData } from '../api/tabs_api'
 
-function setBadgeText (state: RewardsExtension.State): void {
+function setBadgeText (state: RewardsExtension.State, verified: boolean = false, tabId: number = -1): void {
   let text = ''
 
-  if (state && state.notifications) {
+  if (verified) {
+    text = '✔️'
```

Comment:
@rossmoody can you please see if this check is ok or would you like something else. This is how it looks like:

![image](https://user-images.githubusercontent.com/9574457/56655622-9fe01800-6693-11e9-80e1-e56b09ef169d.png)

Reply:
👍